modules/programs/k9s.nix

```
@@ -162,33 +170,70 @@ in {
       { };
 
     skinFiles = mapAttrs' (name: value:
-      nameValuePair "k9s/skins/${name}.yaml" {
-        source = yamlFormat.generate "k9s-skin-${name}.yaml" value;
-      }) cfg.skins;
+      nameValuePair (if !(isDarwin && !config.xdg.enable) then
+        "k9s/skins/${name}.yaml"
+      else
+        "Library/Application Support/k9s/skins/${name}.yaml") {
+          source = yamlFormat.generate "k9s-skin-${name}.yaml" value;
+        }) cfg.skins;
+
   in mkIf cfg.enable {
     home.packages = [ cfg.package ];
 
     xdg.configFile = {
```

Seems unnecessary to duplicate the condition for each file:
```suggestion
    xdg.configFile = mkIf (!(isDarwin && !config.xdg.enable)) {
```

---

Or perhaps better, introduce a let binding

``` nix
enableXdgConfig = !isDarwin || config.xdg.enable
```

and have

```suggestion
    xdg.configFile = mkIf enableXdgConfig {
```

---

Hey! Thanks for taking the time to review the PR, and provide these suggestions :-) .  I added them as a new commit .